app/metro.config.js

```diff
@@ -18,7 +17,7 @@ module.exports = (async () => {
     },
     resolver: {
       assetExts: assetExts.filter(ext => ext !== 'svg'),
-      sourceExts: [...sourceExts, 'svg'],
+      sourceExts: [...sourceExts, 'svg', 'cjs', 'json'],
```

Comment:
What is this line changing here?  Do we need it for this PR?

Reply:
Oh no, sorry! There was a bit in the build that wasn't working, I can't remember which bit but i saw a github that said this might make it work and completely forgot to remove it afterwards. I'll remove that now, and gonna make a quick change form Joanna's notes which she got back to me about this morning (switch out Scotland for UK), plus add a JSDocs comment, and then push it up again